<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Akshay" failures="1" tests="2" name="Test practice(failed)" time="1.878" errors="0" timestamp="2021-06-09T13:13:15 IST">
  <testcase classname="testng1.loginTest" name="login1" time="0.768">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testng1.loginTest.login1(loginTest.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- login1 -->
  <testcase classname="testng1.loginTest" name="login2" time="0.0">
    <skipped/>
  </testcase> <!-- login2 -->
  <testcase classname="testng1.loginTest" name="login" time="0.0">
    <ignored/>
  </testcase> <!-- login -->
</testsuite> <!-- Test practice(failed) -->
